routes/v1.js

```diff
@@ -766,6 +772,7 @@ api.declare({
   method:     'post',
   route:      '/task/:taskId/rerun',
   name:       'rerunTask',
+  stability:  base.API.stability.experimental,
```

Comment:
I guess the difference is that there will only be one resolution of "completed" if we did not have the rerun endpoint.